### PR TITLE
Mirror jenkins

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@bparees

File: cluster/ci/config/mirroring/origin_v4_0_quay [hunk 1]
```diff
@@ -55,6 +55,9 @@ registry.svc.ci.openshift.org/openshift/origin-v4.0:haproxy-router quay.io/opens
...
 registry.svc.ci.openshift.org/openshift/origin-v4.0:hypershift quay.io/openshift/origin-hypershift:v4.0 quay.io/openshift/origin-hypershift:v4.0.0 quay.io/openshift/origin-hypershift:latest
 registry.svc.ci.openshift.org/openshift/origin-v4.0:installer quay.io/openshift/origin-installer:v4.0 quay.io/openshift/origin-installer:v4.0.0 quay.io/openshift/origin-installer:latest
+registry.svc.ci.openshift.org/openshift/origin-v4.0:jenkins quay.io/openshift/origin-jenkins:v4.0 quay.io/openshift/origin-jenkins:v4.0.0 quay.io/openshift/origin-jenkins:latest
+registry.svc.ci.openshift.org/openshift/origin-v4.0:jenkins-maven-agent quay.io/openshift/origin-jenkins-maven-agent:v4.0 quay.io/openshift/origin-jenkins-maven-agent:v4.0.0 quay.io/openshift/origin-jenkins-maven-agent:latest
+registry.svc.ci.openshift.org/openshift/origin-v4.0:jenkins-nodejs-agent quay.io/openshift/origin-jenkins-nodejs-agent:v4.0 quay.io/openshift/origin-jenkins-nodejs-agent:v4.0.0 quay.io/openshift/origin-jenkins-nodejs-agent:latest
```
Comment (bparees) [inline cluster/ci/config/mirroring/origin_v4_0_quay]: can we mirror the base image too?

Comment (bparees) [reply cluster/ci/config/mirroring/origin_v4_0_quay]: jenkins-slave-base

Comment (smarterclayton) [reply cluster/ci/config/mirroring/origin_v4_0_quay]: to mirror the base image you have to be not optional.  who would consume it?

Comment (bparees) [reply cluster/ci/config/mirroring/origin_v4_0_quay]: users who want to build on top of it.  it's not part of the payload, but i'd like to stop manually publishing this stuff to dockerhub, and i saw this as a way to accomplish that.

Comment (bparees) [reply cluster/ci/config/mirroring/origin_v4_0_quay]: made it not optional here:
https://github.com/openshift/release/pull/2662

**Pull request merged**